contracts/VotingAdapter.vy

```diff
@@ -0,0 +1,208 @@
+# @version 0.3.7
+
+"""
+@title Voting Adapter
+@author Lido Finance
+@license MIT
+@notice Used to allow voting with tokens under vesting
+"""
+
+from vyper.interfaces import ERC20
+
+
+interface IDelegation:
+    def setDelegate(
+        _id: bytes32,
+        _delegate: address,
+    ): nonpayable
+
+
+interface IVoting:
+    def vote(
+        _voteId: uint256,
+        _supports: bool,
+        _executesIfDecided_deprecated: bool,
+    ): nonpayable
+
+
+event ERC20Recovered:
+    token: address
+    amount: uint256
+
+
+event ETHRecovered:
+    amount: uint256
+
+
+event OwnerChanged:
+    owner: address
+
+
+ZERO_BYTES32: constant(
+    bytes32
+) = 0x0000000000000000000000000000000000000000000000000000000000000000
+
+VOTING_CONTRACT_ADDR: immutable(address)
+SNAPSHOT_DELEGATE_CONTRACT_ADDR: immutable(address)
+DELEGATION_CONTRACT_ADDR: immutable(address)
```

Comment:
Let's remove this one? since we cannot change it in the future since it's immutable

Reply:
Yep. We have it here to save us some time in the future when delegation will be implemented